pkg/operator/ceph/file/mds/spec_test.go

```diff
@@ -35,6 +35,7 @@ import (
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
 )
 
 func testDeploymentObject(t *testing.T, network cephv1.NetworkSpec) (*apps.Deployment, error) {
```

Comment:
In this test, let's make sure that we test that the liveness probe is configured on the container. It's probably good to check that the script is used and that there aren't any empty env vars set as input.

Reply:
Done